containers/iam-reana-sync/add_reana_users.py

```diff
@@ -0,0 +1,14 @@
+import json
+import os
+import subprocess
+
+client_id = os.environ['CLIENT_ID']
+client_secret  = os.environ['CLIENT_SECRET']  
+reana_admin_token = os.environ['REANA_ADMIN_TOKEN']
+
+with open('/home/emails.json', 'r') as file:
+    j = json.load(file)
+for i in j:
+    output = subprocess.check_output([f'flask reana-admin user-create --email {i} --admin-access-token {reana_admin_token}'], shell=True, encoding='utf-8')
```

Comment:
I would implement some basic error handling here and possibly check the list against users already in the system so as not to cause any exceptions and not run on every user all the time.

```suggestion
    try:
        subprocess.check_output([f'flask reana-admin user-create --email {i} --admin-access-token {reana_admin_token}'], shell=True, encoding='utf-8')
    except subprocess.CalledProcessError as e:
        print(e.stderr)
```

Reply:
Ran black on python files. They look better, thanks @goseind ! Also added the error handling in the script that executes the flask command, even if the error is returned anyways so don't know how useful it is, but it at least looks better.